src/main/drivers/gyro_sync.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * gyro_sync.h
+ *
+ *  Created on: 3 aug. 2015
+ *      Author: borisb
+ */
+
+extern bool gyroIsUpdated;
```

Comment:
Was thinking about that, but I thought that I would need it somewhere else as well and this was easy.
Will probably do that.